.stylelintrc.json

```diff
@@ -28,5 +28,5 @@
     ],
     "csstree/validator": true
   },
-  "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css"]
+  "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css", "**/builds/application.css" ]
```

Comment:
- [ ] I am having an error when I open an app please address it 
![view](https://user-images.githubusercontent.com/44745855/172912098-277e0503-154a-4671-8aa3-2acdbe09c5d8.png)

Reply:
Hi @DeliceLydia,
Thank you for your comments.

 Did you run `yarn` and `bundle install` at the root folder before serving it?